examples/tf/mnist_submission_script.slurm

```diff
@@ -10,6 +10,7 @@
 #SBATCH --time=3:00:00              # maximum execution time (HH:MM:SS)
 #SBATCH --output=tf_mnist%j.out # output file name
 #SBATCH --error=tf_mnist%j.out  # error file name
+#SBATCH --qos=qos_gpu-t4         # it's a testy job
```

Comment:
What do you mean by testy (you know more than me the different queue on Jean Zay) ?
![image](https://user-images.githubusercontent.com/1680079/78906284-71fc0b00-7a7f-11ea-8519-6104492ad9a9.png)

Reply:
no I didn't mean that haha, more test-y like relating to tests in an informal fashion, Will change to something better suited indeed.

Reply:
how does that work for you?